templates/careers.hbs

```diff
@@ -0,0 +1,19 @@
+{{#extend "page"}}
+{{#content "main"}}
+  <h1>{{title}}</h1>
+  {{{contents}}}
```

Comment:
I'd suggest to wrap around the `{{{contents}}}` and the paragraph in a `<section>`. if there are no jobs loaded it won't look half-terrible. ;)